src/vehicle.cpp

```diff
@@ -6351,19 +6374,38 @@ bool vehicle_part::fault_set( const fault_id &f )
     return true;
 }
 
-int vehicle_part::wheel_area() const
+/** Get wheel area or return 0 if part is not wheel */
+float vehicle_part::wheel_area() const
+{
+    return base.is_wheel() ? wheel_diameter() * wheel_width() : 0;
+}
+
+/** Get wheel radius or return 0 if part is not wheel */
+float vehicle_part::wheel_radius() const
```

Comment:
Why keep both `wheel_diameter()` and `wheel_radius()` around simultaneously? I'd leave only one.